plugin/trino-hive-hadoop2/src/test/java/io/trino/plugin/hive/metastore/cache/TestCachingHiveMetastoreWithQueryRunner.java

```diff
@@ -136,6 +136,26 @@ public void testFlushHiveMetastoreCacheProcedureCallable()
         queryRunner.execute(renamedColumnQuery);
     }
 
+    @Test
+    public void testIllegalFlushHiveMetastoreCacheProcedureCalls()
+    {
+        String illegalParameterMessage = "Illegal parameter set passed. ";
```

Comment:
Inline this one, keep the `validUsageExample` as a var.

Reply:
👍 Done